src/lib/components/BaseStateElement/index.js

```diff
@@ -24,7 +24,7 @@ export class BaseStateElement extends BaseElement {
   }
 
   /**
-   * @param {!Object<string, *>} state
+   * Required for `constructor`.
    */
-  onStateChanged(state) {}
+  onStateChanged() {}
```

Comment:
I was wondering if we can add all of it back?

```
* @param {!Object<string, *>} state
*/
onStateChanged(state) {}
```

I've seen folks using this class not know that the state object gets passed to this function, so they end up calling `this.store.getState()` when they don't need to.

Reply:
It doesn't do anything though. I'm not sure what it's supposed to look like or do 😬 (linter doesnt like unused arguments)

Reply:
responded in chat